LayoutTests/fast/text/international/bdo-bidi-width.html

```diff
@@ -1,4 +1,5 @@
 <!DOCTYPE html>
+<meta name="fuzzy" content="maxDifference=192; totalPixels=286-369" />
```

Comment:
this looks like a lot!

Reply:
@alanbaradlay - found the fix. :-) This test was added in [259172@main](https://commits.webkit.org/259172@main) and it was based on the [Blink's commit](https://chromium.googlesource.com/chromium/blink/+/fe79ee4457111490064d8bf9b6c5dce97f26b5d4). Although the blink commit used `isOverride`, which covers both `override` and `isolate-override` but our patch was only for `override`, so when I manually updated locally to have `OR` between then it works similar to Safari 18.2. So I will adjust it in this as well. :-)

Reply:
Fixed. :-)